pyproject.toml

```diff
@@ -1,6 +1,39 @@
 [build-system]
-requires = ['setuptools >= 40.8.0', 'wheel']
-build-backend = 'setuptools.build_meta'
+requires = ["flit_core >=3.3"]
+build-backend = "flit_core.buildapi"
+
+
+[project]
+name = "packaging"
+description = "Core utilities for Python packages"
+version = "21.4.dev0"
+readme = "README.rst"
+requires-python = ">=3.7"
+authors = [{name = "Donald Stufft", email = "donald@stufft.io"}]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "License :: OSI Approved :: BSD License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+urls.Source = "https://github.com/pypa/packaging"
+dependencies = [
+  "pyparsing>=2.0.2,!=3.0.5",
+]
+
+
+[tool.flit.sdist]
+include = ["LICENSE*", "tests/", "docs/"]
+exclude = ["docs/_build", "tests/manylinux/build-hello-world.sh", "tests/musllinux/build.sh", "tests/hello-world.c", "tests/__pycache__", "build/__pycache__"]
```

Comment:
This does not include everything that MANIFEST.in has. Also MANIFEST.in is not removed. I think it's not anymore used with flit?

Reply:
I think the differences are explained above (unless you see something specific). If I forgot MANIFEST.in, that's a mistake. :)

Reply:
https://github.com/pypa/packaging/pull/546#discussion_r870995960

Reply:
@henryiii I just tried to test your branch to see if my change to flit helps. I'm confused. Running nox for tests, docs or lint does not create `tests/__pycache__` or `build/__pycache` for me. Are you sure these are really relevant to exclude anymore with flit?

Reply:
> ... does not create `tests/__pycache__`

I don't know why that would be unless you are running Python with `-B` or the equivalent env var.

> or `build/__pycache__`

I think that was added by mistake, we are not bundling a `build` folder.

Reply:
I was not running Python explicitly. I was running this project's test runner and it did not create `tests/__pycache__`

Reply:
I suggest confirming whether build is necessary and removing tests in follow-up when flit is fixed to exclude bytecode automatically and this project depends on newer version of flit.